src/buffer/out/textBufferCellIterator.cpp

```diff
@@ -126,7 +126,7 @@ TextBufferCellIterator& TextBufferCellIterator::operator+=(const ptrdiff_t& move
     const auto oldX = _pos.X;
     const auto oldY = _pos.Y;
 
-    // Under MSVC writing the individual members of a COORD generates worse assembly
+    // Under MSVC writing the individual members of a til::point generates worse assembly

```

Comment:
> Under MSVC writing the individual members of a til::point generates worse assembly [](http://example.com/codeflow?start=6&length=83)

Maybe not true anymore?

Reply:
It should still be true. MSVC seems to lack a bit in this function and doesn't understand that it can just allocate registers for these operations and only write the result to memory once at the end of the function. Instead it insists on writing it back to memory on each operation which is slow and inhibits optimizations. I hope that I can remove the function entirely in the future though.